architecture/index.md

```diff
@@ -44,6 +44,60 @@ Konflux is a platform for building integrated software that streamlines, consoli
 
 ## Application Context
 
+```mermaid
+graph TD
+    subgraph tenant[Tenant Namespace]
+        App[Application] --> Comp["Component(s)"]
+        App --> Snap["Snapshot(s)"]
+        App --> ITS["IntegrationTestScenario(s)"]
+        App --> RP["ReleasePlan(s)"]
+
+        Comp -- builds --> Snap
+        ITS -- tests --> Snap
+        Snap -- triggers --> Release["Release(s)"]
+        RP -- defines release for --> Release
+
+        Release -- initiates --> PR["PipelineRun(s) (Tenant)"]
+    end
+
+    subgraph managed[Managed Namespace]
+        RPA["ReleasePlanAdmission(s)"]
+        ECP["EnterpriseContractPolicy(s)"]
+
+        Release -- initiates --> PRM["PipelineRun(s) (Managed)"]
+        RPA -- accepts release from --> RP
+        PRM -- enforces policy via --> ECP
+    end
+
+    style App fill:#f9f,stroke:#333,stroke-width:2px
+    style Comp fill:#f9f,stroke:#333,stroke-width:2px
+    style Snap fill:#ccf,stroke:#333,stroke-width:2px
+    style ITS fill:#f9f,stroke:#333,stroke-width:2px
+    style RP fill:#f9f,stroke:#333,stroke-width:2px
+    style Release fill:#ccf,stroke:#333,stroke-width:2px
+    style PR fill:#eee,stroke:#333,stroke-width:1px
+    style RPA fill:#f9f,stroke:#333,stroke-width:2px
+    style ECP fill:#f9f,stroke:#333,stroke-width:2px
+    style PRM fill:#eee,stroke:#333,stroke-width:1px
+
+    classDef controlPlane fill:#f9f,stroke:#333,stroke-width:2px;
+    class App,Comp,ITS,RP,RPA,ECP controlPlane;
+
+    classDef dataPlane fill:#ccf,stroke:#333,stroke-width:2px;
+    class Snap,Release dataPlane;
+
+    classDef tekton fill:#eee,stroke:#333,stroke-width:1px;
+    class PR,PRM tekton;
+
+    click App href "https://redhat-appstudio.github.io/docs/api-references/application-environment-api/#application" "Application API Reference"
```

Comment:
These links don't work

should it be https://konflux-ci.dev/docs/reference/kube-apis/ ?

Reply:
Yeah, good catch. Updated in f3e6896.